tmt/steps/discover/shell.py

```diff
@@ -218,8 +225,10 @@ class DiscoverShell(tmt.steps.discover.DiscoverPlugin[DiscoverShellData]):
     Use provided list of shell script tests.
 
     List of test cases to be executed can be defined manually directly
-    in the plan as a list of dictionaries containing test name, actual
-    test script and optionally a path to the test. Example config:
+    in the plan as a list of dictionaries containing ``test name``, actual
+    ``test script`` and optionally the ``duration`` or a ``path`` to the test.
```

Comment:
There is no field called "test name" or "test script". There are test fields `name` and `script`, therefore only `name` and `script` should be marked as literals. How about the following?

```
    in the plan as a list of dictionaries containing test ``name``, actual
    test ``script`` and optionally the ``duration`` or a ``path`` to the test.
```

Reply:
Agree, code updated !